website/meta/universe.json

```diff
@@ -20,7 +20,7 @@
             "author_links": {
                 "github": "royashcenazi"
```

Comment:
@bdura Hey again, sorry for forgetting this in the merged PR!
can we add it to include parsigs in the "biomedical" category? 

Thanks!

Reply:
Sure thing.